fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/SlotReference.java

```diff
@@ -179,32 +167,28 @@ public SlotReference withChildren(List<Expression> children) {
     }
 
     public SlotReference withDataType(DataType dataType) {
-        return new SlotReference(exprId, name, dataType, nullable, qualifier, canPushDownPredicate, column);
+        return new SlotReference(exprId, name, dataType, nullable, qualifier, column);
     }
 
     public SlotReference withNullable(boolean newNullable) {
         if (this.nullable == newNullable) {
             return this;
         }
-        return new SlotReference(exprId, name, dataType, newNullable, qualifier, canPushDownPredicate, column);
+        return new SlotReference(exprId, name, dataType, newNullable, qualifier, column);
     }
 
     @Override
     public SlotReference withQualifier(List<String> qualifiers) {
-        return new SlotReference(exprId, name, dataType, nullable, qualifiers, canPushDownPredicate, column);
+        return new SlotReference(exprId, name, dataType, nullable, qualifiers, column);
     }
 
     /** withCommonGroupingSetExpression */
     public Slot withCommonGroupingSetExpression(boolean isCommonGroupingSetExpression) {
         if (isCommonGroupingSetExpression) {
-            boolean canPushDownPredicate = true;
-            return new SlotReference(exprId, name, dataType, nullable, qualifier,
-                    Optional.of(canPushDownPredicate), column);
+            return new SlotReference(exprId, name, dataType, nullable, qualifier, column);
         } else {
             boolean nullable = true;
-            boolean canPushDownPredicate = false;
-            return new SlotReference(exprId, name, dataType, nullable, qualifier,
-                    Optional.of(canPushDownPredicate), column);
+            return new SlotReference(exprId, name, dataType, nullable, qualifier, column);
         }
     }
```

Comment:
you can delete this method and use `withNullable` method

Reply:
done